src/components/Accordion/Accordion.stories.mdx

```diff
@@ -96,12 +101,15 @@ const contentData = [
 
 ## Component Props
 
+<ArgsTable story="Accordion with Controls" />
```

Comment:
Why was this moved around?

Reply:
There was a merge conflict in the file when I merged in development and I resolved it by including the current and incoming changes. Because of that, some pieces went into a code section when they were supposed to be part of the text. I moved this to try and make it right again, but I'll have a second look to get it where it needs to be.